lib/domgen/sync/templates/abstract_master_sync_ejb.java.erb

```diff
@@ -57,17 +57,17 @@ public abstract class <%= data_module.sync.abstract_master_sync_ejb_name %>
 
   protected int count<%= entity.qualified_name.gsub('.','') %>Master( @javax.annotation.Nonnull final java.lang.String mappingSourceCode )
   {
-    return (java.lang.Integer) getEntityManager().createNativeQuery( "SELECT COUNT(*) FROM <%= entity.sync.master_entity.sql.qualified_table_name %> WHERE <%= entity.sync.master_entity.attribute_by_name(:MappingSource).sql.quoted_column_name %> = ?" ).setParameter( 1, mappingSourceCode ).getSingleResult();
+    return ( (java.lang.Long) getEntityManager().createNativeQuery( "<%="SELECT COUNT(*) FROM #{entity.sync.master_entity.sql.qualified_table_name} WHERE #{entity.sync.master_entity.attribute_by_name(:MappingSource).sql.quoted_column_name} = ?".gsub('"','\"')%>" ).setParameter( 1, mappingSourceCode ).getSingleResult() ).intValue();
```

Comment:
Rather than adding `.gsub('"','\"')` all over the place, create a helper method in the `java/helper.rb` class (j_escape_string()) that performs this magic